pipelines/build/common/build_base_file.groovy

```diff
@@ -626,7 +627,8 @@ class Builder implements Serializable {
         try {
             context.timeout(time: pipelineTimeouts.API_REQUEST_TIMEOUT, unit: "HOURS") {
                 // Query the Adopt api to get the "tip_version"
-                def JobHelper = context.library(identifier: 'openjdk-jenkins-helper@master').JobHelper
+                String helperRef = DEFAULTS_JSON['repository']['helper_ref']
```

Comment:
Probably needs to check if helper_ref is blank or not set? then default to master?

Reply:
or can we assume DEFAULTS_JSON must set it?

Reply:
so i added it to the pipeline/defaults.json to master as the default value.
see test : https://ci.adoptopenjdk.net/job/build-scripts-pr-tester/job/build-test/job/openjdk11-pipeline/930/parameters/